code/modules/mob/living/simple_animal/abnormality/aleph/silent_orchestra.dm

```diff
@@ -86,13 +86,13 @@
 			if(3)
 				next_movement_time = world.time + 11.5 SECONDS
 				damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0, WHITE_DAMAGE = 1, BLACK_DAMAGE = 0, PALE_DAMAGE = 0)
-				symphony_damage = 15
+				symphony_damage = 18
 				movement_volume = 3 // No more tinnitus
 				spawn_performer(1, EAST)
 			if(4)
 				next_movement_time = world.time + 23 SECONDS
 				damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1, WHITE_DAMAGE = 0, BLACK_DAMAGE = 0, PALE_DAMAGE = 0)
-				symphony_damage = 10
+				symphony_damage = 12
```

Comment:
These all seem beyond fair to be honest... he really did need more buffing as he's just ANNOYING typically.